packages/forklift-console-plugin/src/modules/Plans/views/list/components/PlanStatusCell/PlanStatusCellLabel.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { PlanSummaryStatus } from 'src/modules/Plans/utils';
+import { useForkliftTranslation } from 'src/utils';
+
+import { Label, Spinner } from '@patternfly/react-core';
+
+interface PlanStatusCellLabelProps {
+  status: PlanSummaryStatus;
+  isLoading: boolean;
+}
+
+export const PlanStatusCellLabel: React.FC<PlanStatusCellLabelProps> = ({ status, isLoading }) => {
+  const { t } = useForkliftTranslation();
+
+  if (isLoading) {
+    return <Spinner size="md" />;
```

Comment:
do we have a Loading component in this project? 
If not, could you please add a Loader component in the top share folder? Probably better to create this loader as HOC with prop isLoading that will render children or spinner -this way we will avoid branching in our components.

Reply:
Sure. I will lean more towards render props on this implementation though.